crates/humanode-runtime/src/lib.rs

```diff
@@ -607,23 +629,59 @@ parameter_types! {
     pub WeightPerGas: Weight = Weight::from_ref_time(20_000);
 }
 
+impl pallet_evm_system::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type AccountId = EvmAccountId;
+    type Index = Index;
+    type AccountData = pallet_evm_balances::AccountData<Balance>;
+    type OnNewAccount = ();
+    type OnKilledAccount = ();
+}
+
+impl pallet_evm_balances::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type AccountId = EvmAccountId;
+    type Balance = Balance;
+    type ExistentialDeposit = ConstU128<500>;
+    type AccountStore = EvmSystem;
+    type DustRemoval = primitives_currency_swap_proxy::SwapUnbalanced<
+        currency_swap::EvmToNativeProxy,
+        TreasuryPot,
+        EvmBalancesPot,
+    >;
```

Comment:
Move to the `currency_swap` mod as well.

We can call this one `TreasuryPotProxy`.

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/0507c03dba0fa004a5ea7e0fda1f654510967ca8